js/app/cart/Cart.jsx

```diff
@@ -186,11 +202,13 @@ class Cart extends React.Component
             </div>
           </div>
         </div>
+        {this.state.sameTopCart ? (<CartPortal><CartTop total={sum} i18n={this.props.i18n}/></CartPortal>):(<div></div>)}
```

Comment:
Change for `{this.state.sameTopCart && (<CartPortal><CartTop total={sum} i18n={this.props.i18n}/></CartPortal>)}` which is more elegant ([ref](https://stackoverflow.com/questions/28258465/how-to-add-logical-if-statement-when-rendering-react-components?answertab=votes#tab-top))